API/testrunner/testrunner.py

```diff
@@ -194,16 +194,26 @@ def run(self, app, device, is_publish):
                     continue
 
                 # 3. Process the result.
-                if bool(int(exitcode)) == test.get('expected-failure', False):
+                if bool(int(result['exitcode'])) == test.get('expected-failure', False):
                     reporter.report_pass(test['name'])
                     testresult['result'] = 'pass'
-                    if not 'n/a' in str(memory):
-                        testresult['memory'] = memory
+
+                    jerry_peak = result['jerry_peak_alloc']
+                    malloc_peak = result['malloc_peak']
+                    total_peak = utils.parse_as_int(jerry_peak) + utils.parse_as_int(malloc_peak)
```

Comment:
I would convert the peak values to integer inside before returning the result in the execute call, as you are converting the peaks to integer here, but the testresult dict will have the jerry/malloc peaks as strings

Reply:
`result['jerry_peak_alloc']` can be a number, or `n/a` if there is no jerry-memstat information to the test. Using `n/a` is more informative than using 0 when there is no memory information. That is the reason why I just read the result value into the jerry_peak and malloc_peak variables.

Reply:
Oh.. I see. I've missed the n/a.